esx_service/vmdk_ops.py

```diff
@@ -521,7 +541,27 @@ def removeVMDK(vmdk_path, vol_name=None, vm_name=None, tenant_uuid=None, datasto
                 retry_count += 1
                 time.sleep(vmdk_utils.VMDK_RETRY_SLEEP)
 
-    # remove succeed, remove infomation of this volume from volumes table
+    return None
+
+# Return error, or None for OK
+def removeVMDK(vmdk_path, vol_name=None, vm_name=None, tenant_uuid=None, datastore=None):
```

Comment:
Please add docstring for  function "removeVMDK" to say that removeVMDK handle the normal volume remove (remove .vmdk file and clean the metadata).